### ISSUE TITLE
[QUESTION] 优化log函数返回值的字符串形式

### ISSUE THREAD

**Issue by Ming-Zhou0201** [SwanHubX/SwanLab#816]
### 🤔 Question description [Please make everyone to understand it]
日志显示乱码，但页面显示的图表数据没问题

![Image](https://github.com/user-attachments/assets/5cbd5e17-b914-43f9-a1ae-747a3bd61eca)

![Image](https://github.com/user-attachments/assets/4f14a90a-c35f-4134-8469-10bb9c73b9f1)

### 🚑 Any additional [like screenshots]

- **SwanLab Version**:
- 0.4.6

- **Platform**:
linux

**Comment by SAKURA-CAT** [comment]
您是直接把log的返回值打印了，log的返回值会被加工成对象。不过您的用法也没问题，这块我优化一下

**Comment by Zeyi-Lin** [comment]
在`v0.4.7`版本中支持此特性，感谢反馈！

升级命令：

```bash
pip install -U swanlab
```

**Comment by Ming-Zhou0201** [comment]
> 能看看，第一张图中，打印{'loss': ..., 'acc': ...}的代码吗

epochs = 10
offset = random.random() / 5
for epoch in range(2, epochs):
    acc = 1 - 2**-epoch - random.random() / epoch - offset
    loss = 2**-epoch + random.random() / epoch + offset
    swanlab.log({"acc": acc, "loss": loss}, step=epoch)

swanlab.finish()

**Comment by SAKURA-CAT** [comment]
> 在`v0.4.7`版本中支持此特性，感谢反馈！
> 
> 升级命令：
> 
> pip install -U swanlab

@Ming-Zhou0201 已修复，可升级至0.4.7
